tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -295,60 +297,93 @@ def set_throughput_streams():
             next_step()
 
             start_time = datetime.utcnow()
-            exe_network = benchmark.import_network(args.path_to_model)
+            exe_network = benchmark.core.import_model(args.path_to_model)
             duration_ms = f"{(datetime.utcnow() - start_time).total_seconds() * 1000:.2f}"
-            logger.info(f"Import network took {duration_ms} ms")
+            logger.info(f"Import model took {duration_ms} ms")
             if statistics:
                 statistics.add_parameters(StatisticsReport.Category.EXECUTION_RESULTS,
                                           [
                                               ('import network time (ms)', duration_ms)
                                           ])
-            app_inputs_info, _ = get_inputs_info(args.shape, args.layout, args.batch_size, args.input_scale, args.input_mean, exe_network.input_info)
-            if batch_size == 0:
-                batch_size = 1
+            app_inputs_info, _ = get_inputs_info(args.shape, args.data_shape, args.layout, args.batch_size, args.input_scale, args.input_mean, exe_network.get_runtime_function().get_parameters())
+            batch_size = get_batch_size(app_inputs_info)
+            if batch_size.is_dynamic and benchmark.api_type == 'sync':
```

Comment:
I suppose dynamism is not available for the compiled model at all, is not it?

Reply:
Why not? `ov::runtime::ExecutableNetwork::export_model(std::ostream &networkModel)` doesn't work for dynamic models?